app/imports/templates/components/bid.js

```diff
@@ -21,7 +21,7 @@ function updateRevealedStatus(template, bid) {
     // Timeout, set revealing as false
     console.log('Timeout, set revealing as false')
     MyBids.update({ _id: bid._id }, { $set: {revealing: false} });
-  }, 15000);
+  }, 60000);
```

Comment:
This will still trigger every time, because nothing cancels it. Is that intentional?